doc/source/development.rst

```diff
@@ -314,4 +316,4 @@ order to build the documentation just call:
 Whenever a change in the documentation is pushed to the git, it will be
```

Comment:
* Remove ~the~
* you usually push to GitHub, Bitbucket, GitLab, or any other (remote) repository and not "git" itself. So better use "... to GitHub".